azure-iot-device/tests/common/pipeline/test_pipeline_stages_base.py

```diff
@@ -3513,47 +3605,37 @@ def test_completes_waiting_connect_ops(
             assert op.original_callback.call_count == 1
             assert op.original_callback.call_args == mocker.call(op=op, error=None)
 
-    @pytest.mark.parametrize(
-        "state",
-        [
-            pipeline_stages_base.ReconnectState.NEVER_CONNECTED,
-            pipeline_stages_base.ReconnectState.CONNECTED_OR_DISCONNECTED,
-        ],
+    @pytest.mark.it(
+        "Changes the state to LOGICALLY_DISCONNECTED if the connection fails with an arbitrary error"
```

Comment:
>arbitrary [](start = 85, length = 9)

all these arbitrary errors are non transient errors right ? for me if the statement mentions non transient error it makes more sense..